packages/binding-websockets/src/ws-server.ts

```diff
@@ -299,6 +274,18 @@ export default class WebSocketServer implements ProtocolServer {
                 // eslint-disable-next-line unused-imports/no-unused-vars
                 const event = thing.events[eventName];
 
+                // Populate forms related to the event
+                for (const address of Helpers.getAddresses()) {
+                    const href = this.scheme + "://" + address + ":" + this.getPort() + path;
```

Comment:
Same as above.

Reply:
This could also be formatted like so for the sake of consistency:

```suggestion
                    const href = `${this.scheme}://${address}:${this.getPort()}${path}`;
```